java.md

```diff
@@ -23,14 +23,24 @@ GC time in some workloads, and avoids contention in net-intensive workloads like
 
 Versions of Corretto11 (>=11.0.12) come with additional enhancements to improve
 performance on workloads with light to moderate lock-contention: improved spin-lock behavior inside the JVM,
-enhanced implementation of `Thread.onSpinWait()` on Graviton2.
+enhanced implementation of `Thread.onSpinWait()` on Graviton.
 
 ### Java JVM Options
-There are numerous options that control the JVM and may lead to better performance. Three that
-have shown large (1.5x) improvements in some Java workloads are eliminating tiered compilation
-and restricting the size of the code cache which allows the Graviton2 cores to better predict
-branches. These are helpful on some workloads but can hurt on others so testing with and without
-them is essential: `-XX:-TieredCompilation -XX:ReservedCodeCacheSize=64M -XX:InitialCodeCacheSize=64M`.
+There are numerous options that control the JVM and may lead to better performance.
+
+- Three flags `-XX:-TieredCompilation -XX:ReservedCodeCacheSize=64M -XX:InitialCodeCacheSize=64M`
+have shown large (1.5x) improvements in some Java workloads.
+The flags are eliminating tiered compilation and are restricting the size of
+the code cache which allows the Graviton cores to better predict branches.
+These are helpful on some workloads but can hurt on others so testing with and without
+them is essential.
+
+- Crypto algorithm AES/GCM used by TLS has been optimized for Graviton.
+On Graviton2 GCM encrypt/decrypt
+[performance improves by 3.5x to 5x](https://github.com/aws/aws-graviton-getting-started/issues/110#issuecomment-989442948).
+The optimization is enabled by default in Corretto and OpenJDK 18 and later.
+The optimization has been backported to Corretto and OpenJDK 11 and 17
```

Comment:
Note that it will also be enabled by default in the next Java 11 and 17 update:

https://github.com/openjdk/jdk17u/commit/85a655b908abf2bf45db90192e4b26820ed43930
https://github.com/openjdk/jdk11u/commit/dbd378c4a11949aaa1b6f2360d60f4cac57ea7a7